include/userlib/syscalls.hpp

```diff
@@ -26,6 +27,10 @@ void subscribe_ipc(const char* name, void (*callback)(int message));
 
 void set_fault_handler(void (*callback)(FaultType));
 
+uint32_t start_vtimer(void (*callback)(), uint32_t microseconds);
```

Comment:
start_timer
on another non trivial note, i think we should have an argument in the callback so user code can identify _which_ timer has expired